pkg/converters/ingress/types/global.go

```diff
@@ -96,6 +96,7 @@ const (
 	GlobalModsecurityTimeoutIdle       = "modsecurity-timeout-idle"
 	GlobalModsecurityTimeoutProcessing = "modsecurity-timeout-processing"
 	GlobalModsecurityTimeoutServer     = "modsecurity-timeout-server"
+	GlobalModsecurityArgs              = "modsecurity-args"
```

Comment:
Use alphabetical order here as well

Reply:
It was fixed: https://github.com/jcmoraisjr/haproxy-ingress/pull/948/files#diff-a5c655e369fc7898f6e63f06b4149ecdc24ddd9b94b7b2c89f86c8b78b49bfca

Reply:
Ack, was looking in a wrong branch here, sorry.